rif/lumino/light-client/use/index.md

```diff
@@ -27,25 +28,22 @@ For more information, please take a look at the Readme indications at [https://g
 
     To start using the SDK, you must first configure the Lumino object. This object is our main interface to interact with the SDK. Lumino must be initialized before being used, invoking the init function:
 
-    ```
+    ```javascript
```

Comment:
Does this work? I remember that for some reason, adding an specific language wasn't working in this markdown implementation.